src/components/KTreeList/KTreeItem.vue

```diff
@@ -1,45 +1,39 @@
 <template>
-  <a
-    class="k-tree-item"
+  <div
+    class="tree-item"
     :class="{
       'not-draggable': disabled,
       'selected': item.selected
     }"
-    :data-testid="`k-tree-item-${item.id}`"
-    href="#"
+    :data-testid="`tree-item-${item.id}`"
     role="button"
     @click.prevent="handleClick"
   >
     <div
       v-if="hasIcon"
-      class="k-tree-item-icon"
-      data-testid="k-tree-item-icon"
+      class="tree-item-icon"
+      data-testid="tree-item-icon"
     >
       <slot name="item-icon">
-        <KIcon
-          :icon="itemIcon"
-          :secondary-color="iconSecondaryColor"
-          :size="KUI_ICON_SIZE_40"
-        />
+        <ServiceDocumentIcon />
```

Comment:
suggestion: can you use `DocumentIcon` or `DocumentListIcon` instead?

Reply:
Tried `DocumentIcon` and I think I like `ServiceDocumentIcon` better. `DocumentListIcon` is for list, not actual items.

![Screenshot 2024-03-13 at 4 37 07 PM](https://github.com/Kong/kongponents/assets/36751160/0e334328-1a45-4b06-b33d-7ea7f8388209)

Reply:
IIRC we want to get rid of the service one

Reply:
How about `FileEmptyIcon`?

Reply:
IMO I like the docs ones better than the empty file - but let's just leave the service one for now 
nbd